src/V3Options.h

```diff
@@ -522,6 +523,8 @@ class V3Options {
     }
     string topModule() const { return m_topModule; }
     string unusedRegexp() const { return m_unusedRegexp; }
+    string waiverOutput() const { return m_waiverOutput; }
+    bool trueWaiverOutput() const { return m_waiverOutput.length() > 0; }
```

Comment:
```suggestion
    bool isWaiverOutput() const { return m_waiverOutput.length() > 0; }
```

(some of my Lisp background showing through :)

Reply:
hehe, okay, was wondering about the syntax, but adopted it then :)